packages/ui/product/styles.ts

```diff
@@ -0,0 +1,23 @@
+import { styled } from 'styled-components';
+
+export const ProductImage = styled.img`
+  height: 200px;
+`;
+
+export const CardWrapper = styled.div`
+  margin: 48px auto 0;
+  width: 300px;
+  height: 600px;
+  // display: flex; TO DO - add this when used for the catalog page
+`;
+
+export const CardDetails = styled.div`
+  margin: 1.25rem 0.1rem;
+`;
+
+export const CardComponent = styled.div`
```

Comment:
rename to `StyledDetailsContainer` or `wrapper`

Reply:
Components renamed